package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "discourse-category-sidebars",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "repository": "https://github.com/discourse/discourse-category-sidebars",
   "author": "Discourse",
+  "license": "MIT",
   "devDependencies": {
-    "eslint-config-discourse": "^3.5.0"
+    "eslint-config-discourse": "latest",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
+    "prettier": "^3.0.3",
+    "prettier-plugin-ember-template-tag": "^1.1.0"
```

Comment:
You can now use `@discourse/lint-configs` instead (see https://github.com/discourse/discourse-gifs/pull/50 for an example of transition)

I'll be updating plugin-skeleton and theme-skeleton shortly.